server/services/sessionService.ts

```diff
@@ -34,6 +37,77 @@ export default class SessionService {
     }
   }
 
+  setupAuthSession = async () => {
+    try {
+      // The auth token has more attributes than the handover token
+      const { name, user_uuid: userUuid }: JwtPayloadExtended = jwtDecode(this.request?.user?.token)
+
+      // @ts-expect-error TODO: do something else here
+      this.request.session.handover = {
+        principal: {
+          identifier: userUuid,
+          displayName: name,
+          accessMode: AccessMode.READ_WRITE, // Use 'scope' values instead of hardcoding?
+          authType: AuthType.HMPPS_AUTH,
+          returnUrl: `${config.apis.hmppsAuth.externalUrl}/sign-in?redirect_uri=${config.domain}/sign-in/hmpps-auth/callback`, // Do something useful here...
```

Comment:
I'm guessing the 'do something useful' is in relation to adding the correct redirect stuff for bouncing the user back to the page they originally got navigated to auth from?